packages/rocketchat-ui-vrecord/package.js

```diff
@@ -0,0 +1,31 @@
+Package.describe({
+	"name": "rocketchat:ui-vrecord",
+	"version": "0.0.1",
+	"description": "Video upload with on the fly recording",
+	"documentation": "README.md"
+});
+
+Package.onUse(function(api) {
+	api.versionsFrom('1.2.1');
+
+	api.use([
+		'mongo',
+		'ecmascript',
+		'templating',
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2754409530/issues/source?bid=3199634&fileBranchId=3358103#l14)